model/src/pyrenew/latent/hospitaladmissions.py

```diff
@@ -171,7 +196,9 @@ def sample(
 
         infection_hosp_rate, *_ = self.infect_hosp_rate_rv(**kwargs)
 
-        infection_hosp_rate_t = infection_hosp_rate.value * latent_infections
+        infection_hosp_rate_t = (
```

Comment:
Can we pick better names here? https://github.com/CDCgov/multisignal-epi-inference/issues/357

Reply:
Even better, got rid of that.